action.yml

```diff
@@ -14,6 +14,8 @@ inputs:
     description: The full name of the repository for which the token will be requested (defaults to the current repository).
   permissions:
     description: The JSON-stringified permissions granted to the token (defaults to all the GitHub app permissions, see https://docs.github.com/en/rest/apps/apps#create-an-installation-access-token-for-an-app).
+  github_url:
```

Comment:
Consider changing this to `github_api_url` to make it clear this is meant to override the `GITHUB_API_URL` and not `GITHUB_SERVER_URL` environment variable.